src/components/Flyout/Flyout.tsx

```diff
@@ -27,6 +27,13 @@ import { Overlay } from "./Overlay";
 export const FLYOUT_DIVIDER_COLOR = "#eaebeb";
 export const FLYOUT_DIVIDER_HEIGHT = "10px";
 
+export enum FlyoutPlacement {
+    Top = "top left",
+    Bottom = "bottom left",
+    Right = "right",
+    Left = "left",
```

Comment:
Is aligning left for Top and Bottom the thing we like to enforce? Do we have a need for right-aligned as well?

Reply:
Top left and bottom left were originally enforced but I added the right position for more positioning flexibility.